docs/UserGuide.md

```diff
@@ -163,7 +149,7 @@ Repo owners can provide the following additional information to RepoSense using
 * which git and GitHub usernames belong to which authors
 * the display of an author
 
-To use this feature, add a `_reposense/config.json` to the root of your repo using the format in the example below ([another example](../_reposense/config.json)) and **commit it** (reason: RepoSense can see committed code only):
+To use this feature, add a `_reposense/config.json` to the root of your repo using the format in the example below ([another example](https://github.com/reposense/RepoSense/blob/master/_reposense/config.json)) and **commit it** (reason: RepoSense can see committed code only):
```

Comment:
Is there a reason we are linking to the `config.json` on master branch of the repo, rather than using a relative link to `config.json` ? (so that we can see the `config.json` in the same branch as this document)

Reply:
The reason why I changed it to master branch is because the markbind project is built within the `docs` folder. I couldn't find a way to access resources that are out of the `docs` folder.

@ang-zeyu is there a workaround for this?

Reply:
I think absolute path is the only way as this is a link outside of the site as far as the documentation site is concerned. The html pages are deployed onto a different branch (i.e., gh-pages) anyway, irrespective of the source branch.